src/test/app/wasm_fixtures/wat/deep_recursion.wat

```diff
@@ -0,0 +1,29 @@
+(module
```

Comment:
Can this file just be in `wasm_fixtures` instead of creating a new `wat` folder?